pom.xml

```diff
@@ -227,6 +227,11 @@
         <artifactId>commons-io</artifactId>
         <version>2.4</version>
       </dependency>
+      <dependency>
```

Comment:
Find another json parser.. this one is compiled for java 8. We need one that is compiled for java 6